src/main.rs

```diff
@@ -390,6 +416,17 @@ fn project_arg() -> Arg {
         .help("The project into which the task will be added")
 }
 
+#[cfg(not(tarpaulin_include))]
+fn task_arg() -> Arg {
```

Comment:
I actually might be dumb today (had very little sleep), sorry if I'm blind and silly.

Where it comes into play? I am under the impression that in task_edit we receive tasks only from prompt.

Reply:
Yeah, I double checked and I don't think that this argument is being used either, remove?

Reply:
Yeah, I added it in the beginning thinking the task could be passed as arg, but the implementation changed. I'll remove it!